language/libra-vm/src/logger.rs

```diff
@@ -0,0 +1,123 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::counters::CRITICAL_ERRORS;
+use libra_crypto::HashValue;
+use libra_logger::{prelude::*, Schema};
+use libra_state_view::StateViewId;
+use libra_types::transaction::Version;
+use move_vm_types::logger::Logger;
+use serde::Serialize;
+
+#[derive(Schema, Clone)]
+pub struct LogSchema {
```

Comment:
this may change a bit shortly depending on what we need.
The connection to the executor and the `StateView` will be there but we may describe things differently in the VM. We'll see...